main_windows.go

```diff
@@ -33,16 +33,16 @@ func checkUseInteractiveMode() (bool, error) {
 		return true, nil
 	}
 
-	isInteractiveSession, err := svc.IsAnInteractiveSession()
+	isWindowsService, err := svc.IsWindowsService()
 	if err != nil {
 		return false, fmt.Errorf("failed to determine if we are running in an interactive session: %w", err)
 	}
-	return isInteractiveSession, nil
+	return !isWindowsService, nil
```

Comment:
I updated this logic because I got a deprecation warning for `IsAnInteractiveSession()`.